services/prompter.js

```diff
@@ -71,23 +71,7 @@ async function Prompter(program, requests) {
     }
   }
 
-  if (isRequested('dbSchema')) {
-    if (process.env.DATABASE_SCHEMA) {
-      envConfig.dbSchema = process.env.DATABASE_SCHEMA;
-    } else {
-      prompts.push({
-        type: 'input',
-        name: 'dbSchema',
-        message: 'What\'s the database schema? [optional]',
-        description: 'Leave blank by default',
-        when: answers => answers.dbDialect !== 'sqlite' && answers.dbDialect !== 'mongodb',
-        default: (args) => {
-          if (args.dbDialect === 'postgres') { return 'public'; }
-          return '';
-        },
-      });
-    }
-  }
+  envConfig.dbSchema = program.schema || process.env.DATABASE_SCHEMA || '';
```

Comment:
In [this epic](https://forest.slite.com/app/channels/-KdBU14ToWBAerCk_HGS/notes/MC6abpg~l5#a83c8ed3), we do not remove prompts. You could do something like this:

```javascript
if (isRequested('dbSchema')) {
    // TODO: Remove DATABASE_SCHEMA environment variable usage in the future major Lumber version.
    envConfig.dbSchema = program.schema || process.env.DATABASE_SCHEMA;
    if (!envConfig.dbSchema) {
      // ... prompt code here.
    }
  }
```

(the TODO part was suggested by @arnaudbesnier in one other PR)